app/views/comments/_comment_date.erb

```diff
@@ -4,4 +4,10 @@
       <%= decorated_comment.readable_publish_date %>
     </time>
   </a>
-</div>
+  <% if decorated_comment.edited_at.present? %>
+    • Edited
+    <span class="hidden m:inline-block">
+      on <time datetime="<%= decorated_comment.edited_at %>"><%= decorated_comment.edited_at.strftime('%b %-d') %></time>
```

Comment:
the value of the `datetime` attribute needs to be similar to https://github.com/forem/forem/blob/f95acef2bc5c9f0c1c5f2301f1114b0d95a4871f/app/views/comments/_comment_date.erb#L3

otherwise it won't work in the web page, so not just `.edited_at` but in UTC and ISO8601 format, you can do something like:

https://github.com/forem/forem/blob/f95acef2bc5c9f0c1c5f2301f1114b0d95a4871f/app/decorators/comment_decorator.rb#L8-L12

but for the edited date by adding a method to that decorator file.

The same goes for the date you display, again you can look at this file up a few lines, the method called is `readable_publish_date`:

```ruby
  def readable_publish_date
    if created_at.year == Time.current.year
      created_at.strftime("%b %e")
    else
      created_at.strftime("%b %e '%y")
    end
  end
```

(this basically shows month and day for dates in the current year or adds the year for dates previous to that)

you can do something similar for the edited date (please keep it in the `CommentDecorator` class unlike that method)

Reply:
@rhymes thank you. I'll take note of that. Its a little difficult for me because I'm not very used to the syntax of erb. But I'll make changes and push for review

Reply:
@rhymes Ive been so busy. I'm sorry. I'll make sure to get this resolved today. I have been on this issue for 29days. Isn't that amazing? I doubt any forem issue ever lasted so long. Hahaha

Reply:
> @rhymes Ive been so busy. I'm sorry. I'll make sure to get this resolved today. I have been on this issue for 29days. Isn't that amazing? I doubt any forem issue ever lasted so long. Hahaha

Ha, no worries here. We have many issues/PRs that go for months so you are all good there @KarenEfereyan. Let us know if you need any more help on getting these changes together. Looks like you have been doing a great job keeping up with all the requests.